CRM/Utils/Check/Component/Schema.php

```diff
@@ -123,9 +123,9 @@ public function checkSmartGroupCustomFieldCriteria() {
       if (empty($group['form_values'])) {
         continue;
       }
-      foreach ($group['form_values'] as $formValues) {
-        if (isset($formValues[0]) && (strpos($formValues[0], 'custom_') === 0)) {
-          [, $customFieldID] = explode('_', $formValues[0]);
+      foreach ($group['form_values'] as $key => $val) {
+        if ((strpos($key, 'custom_') === 0)) {
```

Comment:
just fyi - the `str_starts_with`  function is nicer that this `strpos` construct & our polyfills ensure it won't fail on older php versions

Reply:
yes, agree its better, just tried to keep this change minimal by replacing only variable. But agree its a good replacement and a minor one, so I have updated it accordingly :) Thanks.